core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -39,6 +39,19 @@ function formatStatus(status, type, row) {
     return "Dead"
 }
 
+function formatResourceCells(resources) {
```

Comment:
It seems a little crowded from the screenshot. It would be better if we could split this single resource cell into several row cells by resource type. But I guess this may require more changes ? Maybe, add `\n` after each resource type would be good enough ? Or you're intentionally to do this to occupy less space of the cell(since it brings more unnecessary space for other columns(looks more sparse now) ) ?

Reply:
I was trying to take up less space. the linked PR adds in checkboxes to show columns so we could break up more if its not on by default, but I think at this point we should start with something small and then enhance later as people figure out what is useful.